packages/schema/src/config/dev.ts

```diff
@@ -38,6 +38,12 @@ export default defineUntypedSchema({
      * Template to show a loading screen
      * @type {(data: { loading?: string }) => string}
      */
-    loadingTemplate,
+    loadingTemplate: {
+      $resolve (data: Parameters<typeof loadingTemplate>[0], get) {
```

Comment:
How do I test the changes? Where should I look?

I think the current implementation is wrong as I am not sure how to get the value from the `env`?
Should I use `useRuntimeConfig` somehow?

Sorry for the questions, but I am a bit lost. Thanks for any help!

Reply:
it is definitely tricky. we would have needed to support a custom message/prop (maybe custom handling in packages/ui-templates/lib/render.ts). we'd then need to pass the env variable (or option) -> props/messages, which maybe we could do via a wrapped loadingTemplate in schema.

but ... surprisingly difficult